check/test_test.go

```diff
@@ -656,3 +656,45 @@ func TestCompareOp(t *testing.T) {
 		}
 	}
 }
+
+type mockExiter struct {
+	expectedExitCode int
+	passedExitCode   int
+}
+
+func (me *mockExiter) Exit(code int) {
+	me.passedExitCode = code
+}
+
+func TestToNumeric(t *testing.T) {
+	me := &mockExiter{
+		expectedExitCode: 1,
+	}
+
+	toNumeric("a", "b", me)
+	if me.expectedExitCode != me.passedExitCode {
+		t.Errorf("TestToNumeric - Expected Exit Code %d , but instead got %d", me.expectedExitCode, me.passedExitCode)
+	}
+
+	me = &mockExiter{
+		expectedExitCode: 1,
+	}
+
+	toNumeric("5", "b", me)
```

Comment:
This looks to me as a perfect candidate for a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests), where you could have an anonymous struct which args passed to `toNumeric` method and expected results and errors if any (see also my previous comment about adding 3rd arg to `toNumeric` to facilitate unit testing).

Reply:
Agreed 💯